src/Workspaces/Core/Portable/Classification/ClassifiedSpan.cs

```diff
@@ -30,7 +30,7 @@ public override int GetHashCode()
             return Hash.Combine(this.ClassificationType, this.TextSpan.GetHashCode());
         }
 
-        public override bool Equals(object obj)
+        public override bool Equals(object? obj)
         {
             return obj is ClassifiedSpan &&
```

Comment:
Want to use pattern matching while you're here to eliminate the other cast?

Reply:
I'll leave that for another analyzer